molgenis-data-security/src/main/java/org/molgenis/data/security/EntityIdentity.java

```diff
@@ -0,0 +1,18 @@
+package org.molgenis.data.security;
+
+import org.molgenis.data.Entity;
+import org.molgenis.data.meta.model.EntityType;
+import org.springframework.security.acls.domain.ObjectIdentityImpl;
+
+public class EntityIdentity extends ObjectIdentityImpl
+{
+	public EntityIdentity(EntityType entityType, Entity entity)
```

Comment:
Remove EntityType argument, entity type can be retrieved by calling entity.getEntityType